pkg/database/sms_config.go

```diff
@@ -35,6 +35,10 @@ type SMSConfig struct {
 	TwilioAccountSid string `gorm:"type:varchar(250)"`
 	TwilioAuthToken  string `gorm:"type:varchar(250)"` // secret reference
 	TwilioFromNumber string `gorm:"type:varchar(16)"`
+
+	// SMS Config belongs to exactly one realm.
+	RealmID uint
```

Comment:
Can you move this to the top of the struct? Why do we need both a Realm and RealmID?